File: 0344-reverse-string/0344-reverse-string.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        left = 0
        right = len(s) - 1
        while left < right:
            left_char = s[left]
            s[left] = s[right]
            s[right] = left_char
            left += 1
            right -= 1
        
        return s

File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        int left = 0;
        int right = s.length - 1;
        while (left < right) {
                char leftChar = s[left];
                char rightChar = s[right];
                s[left] = rightChar;
                s[right] = leftChar;
                left += 1;
                right -= 1;
        }
    }
}